### ISSUE TITLE
PMID:32571823 FYPO or PECO

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3784]
1.  add condition + tamoxifen

I want this to say

ccr1 | ccr1+[Overexpression] | FYPO:0002716 | normal vacuole fusion during vegetative growth | Cell growth assay | YPD, standard temperature, |+tamoxifen |   | fig5

i.e the vacuole fusion defects caused by tamoxifen in WT are inhibited by OX ccr1 
(it isn't ideal because I can't make the WT observation,  but in total I think its OK with the taxoxifen sensitivities and other experiments, it makes sense)

2.
Then, either
vegetative cell lysis in response to beta-glucanase (precendent: in response to caffeine)
OR
+ beta-glucanase
as a condition.

I'm never sure which way to do these?

**Comment by mah11** [comment]
I don't think we're completely consistent, which doesn't help :P

use conditions for everything except:
- cell population growth (resistance/sensitivity/normal)
- a smallish set of "during cellular response to X" (e.g. HU, MMS, DNA damage, starvation ... mostly stress or nutritional; could probably do with a review to boot some out -- that caffeine example looks dodgy -- but who has time?)

tl;dr: I would use beta-glucanase in conditions for this one

**Comment by ValWood** [comment]
OK, sorry I keep asking this. My brain is a sieve ATM.

Lots of stuff  that we could do with the right amount of resources!

**Comment by ValWood** [comment]
cell lysis during cellular response to caffeine (FYPO:0001804) 
has only one annotation, so maybe boot this one now?

**Comment by mah11** [comment]
could do ... (relieved the paper wasn't one of mine ;) )
